app/models/food.rb

```diff
@@ -5,5 +5,14 @@ class Food < ActiveRecord::Base
   
   has_attached_file :image, :styles => { :medium => "300x300>" }, :default_url => "/images/:style/missing.png"
   validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
+  
+  def self.search(search)
+    if not search.empty?
+      search_condition = "%" + search + "%"
+      where('name ILIKE ?', search_condition)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.